lib/translator/src/Events/Event.php

```diff
@@ -31,12 +31,14 @@ class Event extends PhpObj {
      */
     public function read(array $opts) {
         $appname = $opts['app']->fullname ?: 'A Moodle site';
+		$subField = get_config('logstore_xapi', 'account') ?: 'id';
```

Comment:
For testing purposes, the `get_config` function should not be used in the Translator. Pass this through the constructor instead.